yarn.lock

```diff
@@ -6584,7 +6584,7 @@ eslint@^8.57.0:
     strip-ansi "^6.0.1"
     text-table "^0.2.0"
 
-esm@agoric-labs/esm#Agoric-built, "esm@github:agoric-labs/esm#Agoric-built":
+esm@agoric-labs/esm#Agoric-built:
```

Comment:
weird how this doesn't update on my machine

Reply:
I am not sure what you mean here. I pulled down this branch and am seeing the same `yarn.lock` changes